plugins/modules/discovery_intent.py

```diff
@@ -131,46 +217,14 @@
         description: Order of protocol (ssh/telnet) in which device connection will be tried. For example, 'telnet' - only telnet - 'ssh,
```

Comment:
This is not so clear to me. We are trying to say that if ssh is failed then use telnet. Can we write below?

```
Description: Determines the order in which device connections will be attempted. Here are the options
 - "telnet" Only telnet connections will be tried.
 - "ssh, telnet" SSH (Secure Shell) will be attempted first, followed by telnet if SSH fails.
```